src/contracts/pods/EigenPod.sol

```diff
@@ -667,20 +669,18 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
          * in the beacon chain as a full withdrawal.  Thus such a validator can prove another full withdrawal, and
          * withdraw that ETH via the queuedWithdrawal flow in the strategy manager.
          */
-            // if the withdrawal amount is greater than the MAX_VALIDATOR_BALANCE_GWEI (i.e. the max amount restaked on EigenLayer, per ETH validator)
-        uint64 maxRestakedBalanceGwei = _calculateRestakedBalanceGwei(MAX_VALIDATOR_BALANCE_GWEI);
-        if (withdrawalAmountGwei > maxRestakedBalanceGwei) {
+            // if the withdrawal amount is greater than the MAX_RESTAKED_BALANCE_GWEI_PER_VALIDATOR (i.e. the max amount restaked on EigenLayer, per ETH validator)
+        if (withdrawalAmountGwei > MAX_RESTAKED_BALANCE_GWEI_PER_VALIDATOR) {
             // then the excess is immediately withdrawable
             verifiedWithdrawal.amountToSend =
-                uint256(withdrawalAmountGwei - maxRestakedBalanceGwei) *
+                uint256(withdrawalAmountGwei - MAX_RESTAKED_BALANCE_GWEI_PER_VALIDATOR) *
                 uint256(GWEI_TO_WEI);
-            // and the extra execution layer ETH in the contract is MAX_VALIDATOR_BALANCE_GWEI, which must be withdrawn through EigenLayer's normal withdrawal process
-            withdrawableRestakedExecutionLayerGwei += maxRestakedBalanceGwei;
-            withdrawalAmountWei = maxRestakedBalanceGwei * GWEI_TO_WEI;
+            // and the extra execution layer ETH in the contract is MAX_RESTAKED_BALANCE_GWEI_PER_VALIDATOR, which must be withdrawn through EigenLayer's normal withdrawal process
+            withdrawableRestakedExecutionLayerGwei += MAX_RESTAKED_BALANCE_GWEI_PER_VALIDATOR;
+            withdrawalAmountWei = MAX_RESTAKED_BALANCE_GWEI_PER_VALIDATOR * GWEI_TO_WEI;
         } else {
             // otherwise, just use the full withdrawal amount to continue to "back" the podOwner's remaining shares in EigenLayer
             // (i.e. none is instantly withdrawable)
-            withdrawalAmountGwei = _calculateRestakedBalanceGwei(withdrawalAmountGwei);
```

Comment:
This change addresses EIG 7

Reply:
Is there a specific regression test for this? If so, which test(s) is it?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/blob/873362f84da939e355f1d9d552b7e6c829198b40/src/test/unit/EigenPodUnit.t.sol#L81C10-L81C10